Client/Frontend/Browser/Helpers/ReadyStateScriptHelper.swift

```diff
@@ -55,13 +55,21 @@ class ReadyStateScriptHelper: TabContentScript {
     self.tab = tab
   }
   
-  class func name() -> String {
-    return "ReadyStateScriptHelper"
-  }
-
-  func scriptMessageHandlerName() -> String? {
-    return "ReadyState_\(UserScriptManager.messageHandlerTokenString)"
-  }
+  static let scriptName = "ReadyState"
+  static let scriptId = UUID().uuidString
+  static let messageHandlerName = "\(scriptName)_\(messageUUID)"
+  static let scriptSandbox: WKContentWorld = .page
+  static let userScript: WKUserScript? = {
+    guard var script = loadUserScript(named: scriptName) else {
+      return nil
+    }
+    return WKUserScript.create(source: secureScript(handlerName: messageHandlerName,
+                                                    securityToken: scriptId,
+                                                    script: script),
+                               injectionTime: .atDocumentStart,
+                               forMainFrameOnly: true,
+                               in: scriptSandbox)
+  }()
 
   func userContentController(_ userContentController: WKUserContentController, didReceiveScriptMessage message: WKScriptMessage, replyHandler: (Any?, String?) -> Void) {
     
```

Comment:
`2022-09-14 16:27:26.425 [Error] [ReadyStateScriptHelper.swift:84] userContentController(_:didReceiveScriptMessage:replyHandler:) > Invalid or Missing security token`

This message fails every time because we compare different security tokens here

https://github.com/brave/brave-ios/blob/d7d5eaad3a0cb924ea7093addf7c54122d99b754/Client/Frontend/Browser/Helpers/ReadyStateScriptHelper.swift#L83

Should probably use here `scriptId` instead of `UserScriptManager.securityTokenString`. Could be the same issue in other handlers

Reply:
Fixed.